projects/l2_pooling/multi_column_convergence.py

```diff
@@ -159,22 +217,40 @@ def runExperiment(args):
 
   # For inference, we will check and plot convergence for each object. For each
   # object, we create a sequence of random sensations for each column.  We will
-  # present each sensation for 3 time steps to let it settle and ensure it
+  # present each sensation for 2 time steps to let it settle and ensure it
```

Comment:
Thanks - that's why I introduced a variable, but forgot to update the comment :-)